### PR TITLE
Updated Piwik lib to 2.15.0 and added autoupdate section

### PR DESCRIPTION
**Pull request by quba**
**Library name:** piwik.js
**Git repository url:** https://github.com/piwik/piwik
**License(s):** GNU GPL v3
**Official homepage:** piwik.org
**Wanna say something? Leave message here:** Added the latest stable version of lib and updated package.json

### CODE REVIEW

File: ajax/libs/piwik/package.json [hunk 1]
```diff
@@ -12,12 +12,20 @@
...
+  "license": "deprecated_GPL-3.0+",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/piwik/piwik.git",
+    "basePath": "",
+    "files": [
+        "piwik.js"
+    ]
+  }
+}
```
Comment (PeterDaveHello) [inline ajax/libs/piwik/package.json]: Need a new line char here

File: ajax/libs/piwik/package.json [hunk 2]
```diff
@@ -12,12 +12,20 @@
   "maintainers": [
     {
       "name": "The Piwik Team",
-      "url": "http://piwik.org/the-piwik-team/"
+      "url": "http://piwik.org/team/"
```
Comment (PeterDaveHello) [inline ajax/libs/piwik/package.json]: The origin url still works, I recommend not to touch it here, or should better use another commit to touch it.

**Comment by quba** [comment]
@PeterDaveHello I've just updated the PR.

**Comment by PeterDaveHello** [comment]
@quba Please squash these commits into only one commit as our document says, thanks.

**Comment by quba** [comment]
@PeterDaveHello done.

**Comment by PeterDaveHello** [comment]
@quba Thanks I'll take a look a it.

File: ajax/libs/piwik/package.json [hunk 3]
```diff
@@ -19,5 +19,13 @@
...
     "url": "https://github.com/piwik/piwik.git"
   },
-  "license": "deprecated_GPL-3.0+"
+  "license": "deprecated_GPL-3.0+",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/piwik/piwik.git",
+    "basePath": "",
+    "files": [
+        "piwik.js"
```
Comment (PeterDaveHello) [inline ajax/libs/piwik/package.json]: wrong indent here.

**Comment by quba** [comment]
PeterDaveHello now all checks have passed.

**Comment by PeterDaveHello** [comment]
Thanks @quba

**Pull request merged**